conans/test/integration/options/options_test.py

```diff
@@ -505,3 +505,69 @@ def configure(self):
         assert "dep2/1.0: SHARED: True!!" in c.out
         assert "dep3/1.0: SHARED: True!!" in c.out
         assert "dep4/1.0: SHARED: True!!" in c.out
+
+
+class TestTransitiveOptionsShared:
+    def test_transitive_options_shared_cli(self):
+        c = TestClient()
+        c.save({"dep/conanfile.py": GenConanfile("dep", "0.1").with_shared_option(False),
+                "pkg/conanfile.py": GenConanfile("pkg", "0.1").with_shared_option(False)
+                                                              .with_requires("dep/0.1"),
+                "app/conanfile.txt": "[requires]\npkg/0.1"})
+        c.run("export dep")
+        c.run("export pkg")
+        c.run("install app --build=missing -o *:shared=True")
+        c.run("list dep*:*")
+        assert "shared: True" in c.out
```

Comment:
Should we be more specific and check one by one?

Reply:
And the same comment for the other tests below.

Reply:
Add better checks to tests